### ISSUE TITLE
Train on my own dataset

### ISSUE THREAD

**Issue by jinyu121** [shepnerd/inpainting_gmcnn#53]
Hi~

I am training on my own dataset. In stage 2, I get this image(up to down: GT, image feed to the generator, result)

![image](https://user-images.githubusercontent.com/7346324/72530454-5f23d100-38aa-11ea-8570-652d75bfc65e.png)

As you can see, a darker mask is still on the image. 

Have you ever experienced that? Or, what should I do next? Wait for a longer time, or optimize some hyper-parameters? (Now the parameters are copied from your code and not changed)

**Comment by shepnerd** [comment]
The posted results did seem problematic. Did the first stage training converge? Could you share the training loss trend of both first and second stage?

**Comment by jinyu121** [comment]
Stage 1:

![image](https://user-images.githubusercontent.com/7346324/72581888-dba2c800-391b-11ea-869a-a5a842ed801e.png)

![image](https://user-images.githubusercontent.com/7346324/72581894-e1001280-391b-11ea-9a3a-a2b554b1f999.png)

![image](https://user-images.githubusercontent.com/7346324/72581900-e9f0e400-391b-11ea-9d2d-0ee5c7ceae67.png)

At last, Stage 1 can get images like this(Up to down: gt, input, output):

![image](https://user-images.githubusercontent.com/7346324/72582317-499bbf00-391d-11ea-869c-cb1b821c1ded.png)

Tensorboard record of Stage2 crashed😂

Seems I should wait for longer at Stage 1?

( 1st time to try GAN... May have tons of questions😂)

**Comment by shepnerd** [comment]
It seems you need to training the first stage longer since you conduct natural scenes inpainting. Besides, you may reverse the channel order of the shown images. It appears in BRG instead of RGB channels.

**Comment by congzia** [comment]
How can I show the tensorboard ? When I  python train.py, It tell me "No dashboards are active for the current data set"
